client/my-sites/checkout/checkout/domain-registration-agreement.jsx

```diff
@@ -93,10 +93,12 @@ class DomainRegistrationAgreement extends React.Component {
 	};
 
 	getDomainsByRegistrationAgreement() {
```

Comment:
Btw, if you check `renderAgreements`, it calls `getDomainsByRegistrationAgreement()` with an argument. We can probably do w/o https://github.com/Automattic/wp-calypso/pull/32091/files#diff-d00e073149230055ce69f654d76e1ee4R85.

Reply:
Good catch.  This was leftover from my prev PR.  Thanks!